grails-app/services/au/org/ala/ecodata/ParatooService.groovy

```diff
@@ -875,7 +875,7 @@ class ParatooService {
         // The project/s for the site will be specified by a subsequent call to /projects
         siteData.projects = []
 
-        Site site = Site.findByExternalId(ExternalId.IdType.MONITOR_PLOT_GUID, siteData.externalId)
+        Site site = Site.findByExternalId(ExternalId.IdType.MONITOR_PLOT_SELECTION_GUID, siteData.externalId)
```

Comment:
is ` siteData.externalId` correct? should this not be ` siteData.externalIds[0].externalId`?

Reply:
Thanks @temi, I've fixed this now.